src/components/BenefitsOfCooperation/BenefitsOfCooperation.tsx

```
@@ -0,0 +1,60 @@
+          decoding="async"
+        />
+      </div>
+      <div className="benefits-desktop">
+        {BENEFITS_DATA.map(item => (
+          <BenefitBlock
+            image={item.image}
+            title={item.title}
+            text={item.text}
+          />
+        ))}
+        <img
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)